lambdas/death-enrichment/src/test/resources/uk/gov/di/data/lep/approvals/DeathEnrichmentIntegrationTest.approvalTestForOutput.approved.txt

```diff
@@ -1 +1 @@
-{"aud":null,"events":{"https://ssf.account.gov.uk/v1/deathRegistration":{"deathDate":{"description":null,"value":"2007-03-06"},"deathRegistrationID":1234567890,"freeFormatDeathDate":null,"deathRegistrationTime":{"value":"2023-03-06T09:30:50.000"},"subject":{"address":[{"addressCountry":null,"addressLocality":null,"buildingName":null,"buildingNumber":null,"departmentName":null,"dependentAddressLocality":null,"dependentStreetName":null,"doubleDependentAddressLocality":null,"organisationName":null,"postalCode":"GT8 5HG","streetName":null,"subBuildingName":null,"uprn":null,"validFrom":null,"validUntil":null}],"birthDate":[{"description":null,"value":"1967-03-06"}],"name":[{"description":null,"nameParts":[{"type":"GivenName","value":"ERICA","validFrom":null,"validUntil":null},{"type":"FamilyName","value":"BLOGG","validFrom":null,"validUntil":null}],"validFrom":null,"validUntil":null}],"sex":["Female"]}}},"exp":null,"iat":1,"iss":null,"jti":"guid_1","nbf":null,"sub":null,"toe":1678095050,"txn":"guid_2"}
\ No newline at end of file
+{"aud":null,"events":{"https://ssf.account.gov.uk/v1/deathRegistration":{"deathDate":{"description":null,"value":"2007-03-06"},"deathRegistrationID":1234567890,"freeFormatDeathDate":null,"deathRegistrationTime":"2023-03-06T09:30:50.000","subject":{"address":[{"addressCountry":null,"addressLocality":null,"buildingName":null,"buildingNumber":null,"departmentName":null,"dependentAddressLocality":null,"dependentStreetName":null,"doubleDependentAddressLocality":null,"organisationName":null,"postalCode":"GT8 5HG","streetName":null,"subBuildingName":null,"uprn":null,"validFrom":null,"validUntil":null}],"birthDate":[{"description":null,"value":"1967-03-06"}],"name":[{"description":null,"nameParts":[{"type":"GivenName","value":"ERICA","validFrom":null,"validUntil":null},{"type":"FamilyName","value":"BLOGG","validFrom":null,"validUntil":null}],"validFrom":null,"validUntil":null}],"sex":["Female"]}}},"exp":null,"iat":1,"iss":null,"jti":"guid_1","nbf":null,"sub":null,"toe":1678095050,"txn":"guid_2"}
```

Comment:
Would potentially expect a Z on the end of this time? Or are we dealing with zones in a separate change?

Reply:
Good shout, I can cover that off here I think

Reply:
Actually, I think given the vocab change is draft and we're pending confirmation from HMPO on what the date incoming is, I think I _won't_ make this change here.